src/app/static/src/tests/components/surveyAndProgram/fileUploads.ts

```diff
@@ -110,25 +124,25 @@ export function testUploadComponent(name: string, position: number) {
 
     it(`${name} upload is valid if data is present`, () => {
         const store = createSut(successState);
-        const wrapper = shallowMount(SurveyAndProgram, {store, localVue});
+        const wrapper = shallowMount(Baseline, {store, localVue});
```

Comment:
These tests should really move into the `components/baseline` folder to test functionality which is now happening from the `Baseline` component

Reply:
done